packages/react-core/src/filters/viewportFeaturesGeoJSON.d.ts

```diff
@@ -1,8 +1,9 @@
-import { FeatureCollection } from 'geojson';
+import { FeatureCollection, Geometry } from 'geojson';
 import { Viewport, ViewportFeaturesResponse } from '../types';
 
 type ViewportFeaturesGeoJSONArgs = {
   geojson: FeatureCollection,
+  spatialFilterGeometry: Geometry,
```

Comment:
spatialFilterGeometry can be null, right? so it should be optional

Reply:
Updated